src/service/viewport-impl.js

```diff
@@ -140,6 +143,16 @@ export class Viewport {
 
     this.binding_.onScroll(this.scroll_.bind(this));
     this.binding_.onResize(this.resize_.bind(this));
+
+    this.onScroll(() => {
+      if (!this.duringAnimationFrame_) {
+        this.duringAnimationFrame_ = true;
+        this.vsync_.measure(() => {
+          this.duringAnimationFrame_ = false;
+          this.viewer_.postScroll(this.binding_.getScrollTop());
```

Comment:
@dvoytenko suggested me to get the value from the binding_. Are we having trouble if we call `this.binding_.getScrollTop()`?